src/lib/ClassPreferences.js

```diff
@@ -0,0 +1,40 @@
+const VERSION = 'v1'; // In case we ever need to invalidate these
+let cache = {};
```

Comment:
I'm unsure how practical this type of ephemeral storage is for filter views. Maybe we could allow to export these filter definitions to copy/paste them into the dashboard config file to keep the filters persistently? This way the dashboard can be used as a convenient view designer, and the user can make the filter permanent through config deployment.

Reply:
Hoping after #2406, we can have a way to export `ClassPreferences` to be defined in the Dashboard config